### PR TITLE
dev/reference/transactions: fix some format mistakes

### PR DESCRIPTION
**Pull request by IzabelWang**
Fix some format mistakes in transaction-pessimistic.md
PTAL @lilin90

### CODE REVIEW

File: dev/reference/transactions/transaction-pessimistic.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ TiDB 默认使用乐观事务模式，存在事务提交时因为冲突而失败
 
 > **注意：**
 >
-> 悲观事务模式目前是一个**试验性**的特性，*不建议在生产环境中使用*。
+> 悲观事务模式目前是一个**试验性**的特性，**不建议在生产环境中使用**。
 
 ## 悲观事务模式的行为
 
-悲观事务的行为和 MySQL 基本一致(不一致的地方详见[已知的局限](#已知的局限))， 
+悲观事务的行为和 MySQL 基本一致(不一致的地方详见[已知的局限](#已知的局限))：
```
Comment (lilin90) [inline dev/reference/transactions/transaction-pessimistic.md]:
```suggestion
悲观事务的行为和 MySQL 基本一致（不一致的地方详见[已知的局限](#已知的局限)）：
```

File: dev/reference/transactions/transaction-pessimistic.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ TiDB 默认使用乐观事务模式，存在事务提交时因为冲突而失败
...
 ```
 [pessimistic-txn]
 enable = true
 ```
 
-`enable` 设置为 `true` 以后，默认的事务模式仍然是乐观事务模式，要进入悲观事务模式有以下三种方式:
+`enable` 设置为 `true` 以后，默认的事务模式仍然是乐观事务模式，要进入悲观事务模式有以下三种方式：
 
 - 执行 `BEGIN PESSIMISTIC;` 语句开启的事务，会进入悲观事务模式。
 可以通过写成注释的形式 `BEGIN /*!90000 PESSIMISTIC */;`来兼容 MySQL 语法。
```
Comment (lilin90) [inline dev/reference/transactions/transaction-pessimistic.md]:
```suggestion
可以通过写成注释的形式 `BEGIN /*!90000 PESSIMISTIC */;` 来兼容 MySQL 语法。
```

File: dev/reference/transactions/transaction-pessimistic.md [hunk 3]
```diff
@@ -61,33 +61,33 @@ default = true
...
 
 - 其次是 session 变量 `tidb_txn_mode`。
 
-- 最后是配置文件里的 `default`, 当使用普通的 `BEGIN` 语句，且 `tidb_txn_mode` 的值为空字符串 '' 时，根据 `default` 来决定启用悲观事务还是乐观事务。  
+- 最后是配置文件里的 `default`，当使用普通的 `BEGIN` 语句，且 `tidb_txn_mode` 的值为空字符串 `''` 时，根据 `default` 来决定启用悲观事务还是乐观事务。  
 
 ## 相关配置参数
 
-相关配置都在 `[pessimistic-txn]` 类别下，除了前面介绍过的 `enable` 和 `default`，还有以下参数可以配置:
+相关配置都在 `[pessimistic-txn]` 类别下，除了前面介绍过的 `enable` 和 `default`，还有以下参数可以配置：
```
Comment (lilin90) [inline dev/reference/transactions/transaction-pessimistic.md]:
```suggestion
相关配置都在 `[pessimistic-txn]` 类别下，除了前面介绍过的 `enable` 和 `default`，还可配置以下参数：
```

File: dev/reference/transactions/transaction-pessimistic.md [hunk 4]
```diff
@@ -61,33 +61,33 @@ default = true
```
Comment (lilin90) [inline dev/reference/transactions/transaction-pessimistic.md]: Line 60 lacks a space.

Comment (IzabelWang) [reply dev/reference/transactions/transaction-pessimistic.md]: OK, addressed.

File: dev/reference/transactions/transaction-pessimistic.md [hunk 5]
```diff
@@ -57,37 +57,37 @@ default = true
...
 - `ttl` 
 
-```
-ttl = "30s"
-```
+    ```
+    ttl = "30s"
+    ```
 
 `ttl` 是悲观事务的锁超时时间，默认值 30 秒，可以在 15 秒到 60 秒之间配置，超出这个范围会报错。
```
Comment (lilin90) [inline dev/reference/transactions/transaction-pessimistic.md]: Please indent line 82 ~ 84 with 4 spaces too. Move them to a same line.

File: dev/reference/transactions/transaction-pessimistic.md [hunk 6]
```diff
@@ -57,37 +57,37 @@ default = true
...
 - `max-retry-count`
 
-```
-max-retry-count = 256
-```
+    ```
+    max-retry-count = 256
+    ```
 
 悲观事务会在事务内自动重试单个语句，这个配置指定单个语句最大重试次数，设置这个参数是为了避免极端情况下，出现无限重试，无法停止。
```
Comment (lilin90) [inline dev/reference/transactions/transaction-pessimistic.md]: ditto

Comment (IzabelWang) [reply dev/reference/transactions/transaction-pessimistic.md]: Thanks~ I will handle format more carefully.

**Pull request merged**